runtime/metricsview/astexpr.go

```diff
@@ -408,10 +408,14 @@ func (b *sqlExprBuilder) writeInCondition(left, right *Expression, leftOverride
 }
 
 func (b *sqlExprBuilder) writeInConditionForValues(left *Expression, leftOverride string, vals []any, not bool) error {
```

Comment:
It will be great if we can include examples as unit tests for various if else cases handled here